fieldservice_route/tests/test_fsm_order.py

```diff
@@ -0,0 +1,22 @@
+# Copyright (C) 2019 Open Source Integrators
+# Copyright (C) 2019 Serpent consulting Services
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl.html).
+
+from odoo.tests.common import TransactionCase
+from datetime import datetime
+
+
+class FSMOrderRouteCase(TransactionCase):
+
+    def setUp(self):
+        super(FSMOrderRouteCase, self).setUp()
+        self.fsm_stage_obj = self.env['fsm.stage']
+        self.fsm_order_obj = self.env['fsm.order']
+        self.test_person = self.env.ref("fieldservice.test_person")
+        self.test_location = self.env.ref("fieldservice.test_location")
+
+    def test_create_day_route(self):
+        self.fsm_order_obj.create({'location_id': self.test_location.id,
+                                   'person_id': self.test_person.id,
+                                   'scheduled_date_start': datetime.now(),
+                                   })
```

Comment:
@nikul-serpentcs Can you add the tests to:

- [ ] check that the route of the order is the same as the route of the location
- [ ] check that the order is added to a dayroute with the same person, date and route

Reply:
Done